Notification: feeding notification from the backend events to front end .

vthakur7f opened the pull request:
Two services  :
- [x] Event notify: Process events from the backend
- [x] Distributor: Post the events on web socket for clients.

salman-codewar reviewed (approved):
@vthakur7f Thanks for working on this. Looks good 💯 

PR have some commented code and some docker compose required changes. But i addressed it on my PR while testing it. So we can merge this one.

vthakur7f commented:
> ome docker compose required changes. But i addressed it on my PR while testing it. So we can merge this one.



> @vthakur7f Thanks for working on this. Looks good 💯
> 
> PR have some commented code and some docker compose required changes. But i addressed it on my PR while testing it. So we can merge this one.

@salman-codewar thanks
